lib/simplifyExpression/collectAndCombineSearch/multiplyLikeTerms.js

```diff
@@ -5,23 +5,20 @@ const multiplyFractionsSearch = require('../multiplyFractionsSearch');
 
 const ChangeTypes = require('../../ChangeTypes');
 const Node = require('../../node');
-
 // Multiplies a list of nodes that are polynomial like terms. Returns a node.
 // The polynomial nodes should *not* have coefficients. (multiplying
 // coefficients is handled in collecting like terms for multiplication)
-function multiplyLikeTerms(node, polynomialOnly=false) {
+function multiplyLikeTerms(node, polynomialOnly = false) {
```

Comment:
From what I remember, the lack of spaces is a special syntax thing to show that it's a default value for `polynomialOnly` instead of the usual use of `=`